WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -603,6 +608,8 @@ class MediaCoordinator: NSObject {
                 return "Failed: \(error)"
             case .progress(let value):
                 return "Progress: \(value)"
+            case .cancelled:
+                return "Cancelled"
```

Comment:
[nit] Let's use the US spelling for canceled
```suggestion
        case canceled
```
```suggestion
            case .canceled:
                return "Canceled"
```